src/backend/distributed/executor/multi_utility.c

```diff
@@ -185,23 +184,29 @@ multi_ProcessUtility(Node *parsetree,
 	Oid savedUserId = InvalidOid;
 	int savedSecurityContext = 0;
 	List *ddlJobs = NIL;
-	bool skipCitusProcessing = SkipCitusProcessingForUtility(parsetree);
```

Comment:
I removed this function, because in the version we error out almost all queries, we added this to whitelist some of the DDLs like CREATE/ALTER/DROP extension... Now since we do not error out non-distributed queries, we do not need this.